src/components/Avatar/Avatar.module.css

```diff
@@ -131,3 +131,9 @@ button.avatar:disabled {
 .stacked-avatars .avatar:not(:first-child) {
   margin-left: calc(var(--cpd-avatar-size) * -0.2);
 }
+
+.clip-path {
+  /* The SVG is invisible, but we still need to ensure it doesn't affect the
+  page's layout */
+  position: fixed;
```

Comment:
`left: -9999px` too for good measure? Feel like it might not always be transparent?

Reply:
Sure, that wouldn't hurt